op-batcher/metrics/metrics.go

```diff
@@ -55,8 +58,10 @@ type Metrics struct {
 	// label by openend, closed, fully_submitted, timed_out
 	channelEvs opmetrics.EventVec
 
-	pendingBlocksCount prometheus.GaugeVec
-	blocksAddedCount   prometheus.Gauge
+	pendingBlocksCount        prometheus.GaugeVec
+	pendingBlocksBytesCount   prometheus.Counter
+	pendingBlocksBytesCurrent prometheus.Gauge
+	blocksAddedCount          prometheus.Gauge
```

Comment:
I realize that it was a confusing idea to end scalar gauges in `Count` because there's also counters... I like using `Current`. However, I propose we keep it consistent within the file and with the general prometheus convention to suffix counters with `Total`.
```suggestion
	pendingBlocksCount        prometheus.GaugeVec
	pendingBlocksBytesTotal   prometheus.Counter
	pendingBlocksBytesCurrent prometheus.Gauge
	blocksAddedCount          prometheus.Gauge
```